src/utils/planner.ts

```diff
@@ -0,0 +1,10 @@
+import { PlannerPlan } from "@microsoft/microsoft-graph-types";
+import { odata } from "./odata";
+
+const graphResource = 'https://graph.microsoft.com';
+
+export const planner = {
+  async getPlansByGroupId(groupId: string): Promise<PlannerPlan[]> {
+    return odata.getAllItems<PlannerPlan>(`${graphResource}/v1.0/groups/${groupId}/planner/plans`, undefined as any);
```

Comment:
By using the `odata.getAllItems<T>` utility, I noticed that this method expects a `logger` parameter which is not used in the function itself. This is the reason why I pass `undefined as any` as logger parameter in my planner utility.

https://github.com/pnp/cli-microsoft365/blob/99da27d7a47c48eae97db6a6631d4826fdf1a5de/src/utils/odata.ts#L21-L43

Maybe an idea to remove this logger parameter from the odata.getAllItems utility?

Reply:
If we don't use the logger, then we should definitely remove it from the method's signature. Good catch @milanholemans 👍

Reply:
Let's do this in PR; it'll help us keep it all in one place and avoid adding code that we'll be removing.

Reply:
Made an extra commit to remove the unused parameter.